app/src/main/res/values-da/strings.xml

```diff
@@ -245,4 +257,6 @@
     <string name="faq_5_text">Tjek batteriet på din enhed og indstillingerne for meddelelser, om der er noget, der blokerer påmindelserne, eller om appen er slået fra i baggrunden. Du kan også tage et kig på &lt;a href=https://dontkillmyapp.com&gt;https://dontkillmyapp.com&lt;/a&gt;, den har nogle gode tips.</string>
     <string name="faq_6_title">Hvordan kan jeg ændre eller slette en eksisterende begivenhedstype\?</string>
     <string name="faq_6_text">Du kan gøre begge dele i Indstillinger - Administrer begivenhedstyper. Du skal blot klikke på den ønskede begivenhed for at ændre etiketten og farven, eller du kan vælge den ønskede begivenhed ved at trykke længe på den og bruge papirkurven i topmenuen til at slette den.</string>
-</resources>
\ No newline at end of file
+    <string name="package_name">org.fossify.calendar</string>
+    <string name="app_name">Fossify Kalender</string>
```

Comment:
Please don't translate these strings. They are from `donottranslate.xml`.

Reply:
Are you really sure you want app_name in donottranslate.xml?

Reply:
@lianergoist yes.